docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+`     PRECHECKBASE:1,Fortitude=5,Reflex=3    `
+
+Would succeed if Fortitude meets or exceeds 5 or Reflex meets or exceeds 3.
+
+------------------------------------------------------------------------
+
+**<span id="precity"></span> Tag Name:** PRECITY:x
+
+**Variables Used (x):** Text (The name of a city that you must live in).
+
+**What it does:**
+
+Sets the required city. A character's city is set in the [Description Tab](../../tabpages/tabdescription.html)
+
+**Example:**
+
+`     PRECITY:Klamath    `
+
+Character must currently reside in "Klamath".
+
+------------------------------------------------------------------------
+
+**<span id="preclass"></span> Tag Name:** PRECLASS:x,y=z,y=z
+
+**Variables Used (x):** Number (The number of classes that must be equal to or greater than the numbers specified for the check to succeed).
+
+**Variables Used (y):** Text (Class Name)
+
+**Variables Used (y):** TYPE.Text (Class Type)
+
+**Variables Used (y):** SPELLCASTER.
+
+**Variables Used (y):** SPELLCASTER.Type (Spellcaster Type)
+
+**Variables Used (y):** ANY
+
+**Variables Used (z):** Number (Class Level)
+
+**What it does:**
+
+Sets class requirements.
+
+**Example:**
+
+`     PRECLASS:2,Wizard=5,Sorcerer=6,Cleric=7    `
+
+Multi-classed character must be at least Wiz5/Sor6, Wiz5/Clr7 or Sor6/Clr7.
+
+`     PRECLASS:1,SPELLCASTER=2    `
+
+Character must have 2 levels in any spellcasting class. This encompasses psionic manifesting classes as well, as PCGen does not treat manifesting differently from spellcasting. Spellcaster and manifester can be considered interchangeable.
+
+`     PRECLASS:1,SPELLCASTER.Arcane=2    `
+
+Character must have 2 levels in any arcane spellcasting class.
+
+`     PRECLASS:1,SPELLCASTER.Divine=6,SPELLCASTER.Psionic=3    `
+
+Character must have 6 levels in any divine spellcasting class or 3 levels in any psionic manifesting class.
+
+`     PRECLASS:2,TYPE.Base=5,TYPE.Prestige=1    `
+
+Character must have 5 levels in any Base class and 1 level in any Prestige class.
+
+`     PRECLASS:2,ANY    `
+
+Multi-classed character must be at least two Classes.
+
+`     PRECLASS:1,ANY    `
+
+Character must have 1 level in any class.
+
+`     PREMULT:1,[PREFEAT:1,Blood of the Fey],[PRECLASS:1,Fey-Touched=1]    `
+
+Character must have 1 level in Fey-Touched class or feat Blood of the Fey.
+
+`     PRECLASS:1,Jack o' the Green=5 !PREFEAT:1,TYPE.LASlipperyEel    `
+
+Character must have 5 levels in Jack o' the Green class and not have the feat LASlipperyEel.
+
+`     !PRECLASS:1,SPELLCASTER.Arcane=1    `
+
+Character must not have 1 level in SPELLCASTER.Arcane class.
+
+`     BONUS:COMBAT|AC|1|PRECLASS:1,Shaper=10    `
+
+Character adds 1 to armor class if 10 levels in Shaper Class.
+
+`     BONUS:WEAPON|TOHIT|4|TYPE=Luck|PRECLASS:1,Wizard=1,Sorcerer=1    `
+
+Character adds 4 to To-Hit from luck if 1 level in either Wizard/Sorcerer Class.
+
+`     BONUS:CASTERLEVEL|Minstrel|CL-3|PRECLASS:1,Minstrel=4    `
```

Comment:
Yikes! "CL-3" will not work out of context. This should be noted as embedded in a CLASS file. Or change it to classlevel("Minstrel")-3.